analyzers/PDebugger.py

```diff
@@ -0,0 +1,30 @@
+from heppy.framework.analyzer import Analyzer
+import logging
+import heppy.utils.pdebug as pdebugging
+from heppy.papas.data.pfevent import PFEvent
+import sys
+
+
+class PDebugger(Analyzer):
+    ''' Module to output physics debug output
+    '''
+    def __init__(self, *args, **kwargs):
+        super(PDebugger, self).__init__(*args, **kwargs)
+
+        #no output will occur unless one or both of the following is requested.
+
+        #turn on output to stdout if requested
+        if hasattr(self.cfg_ana, 'output_to_stdout') and self.cfg_ana.output_to_stdout:
+                pdebugging.set_stream(sys.stdout,level=logging.INFO)
+                pdebugging.pdebugger.setLevel(logging.INFO)
+
+        #turn on output to file if requested
+        if hasattr(self.cfg_ana, 'debug_filename'):
+            pdebugging.set_file(self.cfg_ana.debug_filename)
+            pdebugging.pdebugger.setLevel(logging.INFO)
+        pass
+
+    def process(self, event):
```

Comment:
not clear what this analyzer is doing. from the process method, i see that it only prints the event number, but the docstrings seem to imply that it's much more than that. 
The documentation should explain why this analyzer is necessary. 
For instance, mention that this analyzer: 
- sets up the pdebugging tool, a separate logger for physics
- logs for each event the number of event
Then, explain that the pdebugging module should be used wherever the user wants a physics debug output.
no need to repeat the pdebugging documentation, the documentation for this module should be therein